src/Configurator/EnvConfigurator.php

```diff
@@ -51,7 +51,11 @@ private function configureEnvDist(Recipe $recipe, $vars, bool $update)
             foreach ($vars as $key => $value) {
                 $value = $this->evaluateValue($value);
                 if ('#' === $key[0] && is_numeric(substr($key, 1))) {
-                    $data .= '# '.$value."\n";
+                    if ('' === $value) {
```

Comment:
```suggestion
                    if ('' === trim($value)) {
```

WDYT @fabpot ?

Reply:
this won't happend IRL, and if it does, I would trust the manifest did it on purpose

Reply:
OK, ready to merge then